sdk/openai/ci.yml

```diff
@@ -0,0 +1,33 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/openai/ci.yml
```

Comment:
Is there a specific reason not to track `sdk/openai`? Otherwise changes made to that directory will likely need a check enforcer override.